Exception in plugin Rust

Was working on pretty small file and got error:
```
Assertion failed
java.lang.AssertionError: Assertion failed
	at org.rust.ide.formatter.impl.AlignmentKt$getAlignmentStrategy$1.invoke(alignment.kt:28)
	at org.rust.ide.formatter.impl.AlignmentKt$getAlignmentStrategy$1.invoke(alignment.kt)
	at org.rust.ide.formatter.RustAlignmentStrategy$alignIf$2.getAlignment(RustAlignmentStrategy.kt:45)
	at org.rust.ide.formatter.RustAlignmentStrategy$alignIf$2.getAlignment(RustAlignmentStrategy.kt:46)
	at org.rust.ide.formatter.blocks.RustFmtBlock.buildChildren(RustFmtBlock.kt:60)
	at org.rust.ide.formatter.blocks.RustFmtBlock.access$buildChildren(RustFmtBlock.kt:14)
	at org.rust.ide.formatter.blocks.RustFmtBlock$mySubBlocks$2.invoke(RustFmtBlock.kt:29)
	at org.rust.ide.formatter.blocks.RustFmtBlock$mySubBlocks$2.invoke(RustFmtBlock.kt:14)
	at kotlin.SynchronizedLazyImpl.getValue(Lazy.kt:131)
	at org.rust.ide.formatter.blocks.RustFmtBlock.getMySubBlocks(RustFmtBlock.kt)
	at org.rust.ide.formatter.blocks.RustFmtBlock.getSubBlocks(RustFmtBlock.kt:28)
	at com.intellij.formatting.InitialInfoBuilder.buildFrom(InitialInfoBuilder.java:227)
	at com.intellij.formatting.InitialInfoBuilder.doIteration(InitialInfoBuilder.java:326)
	at com.intellij.formatting.InitialInfoBuilder.iteration(InitialInfoBuilder.java:152)
	at com.intellij.formatting.engine.WrapBlocksState.doIteration(WrapBlocksState.java:47)
	at com.intellij.formatting.engine.State.iteration(State.java:25)
	at com.intellij.formatting.engine.StateProcessor.iteration(StateProcessor.java:41)
	at com.intellij.formatting.FormatProcessor.iteration(FormatProcessor.java:98)
	at com.intellij.formatting.FormatterImpl.buildProcessorAndWrapBlocks(FormatterImpl.java:493)
	at com.intellij.formatting.FormatterImpl.buildProcessorAndWrapBlocks(FormatterImpl.java:479)
	at com.intellij.formatting.FormatterImpl.adjustLineIndentsForRange(FormatterImpl.java:372)
	at com.intellij.psi.impl.source.codeStyle.CodeStyleManagerImpl$4.doPerform(CodeStyleManagerImpl.java:390)
	at com.intellij.psi.impl.source.codeStyle.CodeStyleManagerRunnable.perform(CodeStyleManagerRunnable.java:109)
	at com.intellij.psi.impl.source.codeStyle.CodeStyleManagerImpl.adjustLineIndent(CodeStyleManagerImpl.java:393)
	at com.intellij.codeInsight.editorActions.enter.EnterAfterUnmatchedBraceHandler.formatCodeFragmentBetweenBraces(EnterAfterUnmatchedBraceHandler.java:226)
	at com.intellij.codeInsight.editorActions.enter.EnterAfterUnmatchedBraceHandler.insertRBraces(EnterAfterUnmatchedBraceHandler.java:176)
	at com.intellij.codeInsight.editorActions.enter.EnterAfterUnmatchedBraceHandler.preprocessEnter(EnterAfterUnmatchedBraceHandler.java:67)
	at com.intellij.codeInsight.editorActions.EnterHandler.executeWriteActionInner(EnterHandler.java:126)
	at com.intellij.codeInsight.editorActions.EnterHandler.lambda$executeWriteAction$0(EnterHandler.java:79)
	at com.intellij.psi.impl.source.PostprocessReformattingAspect.lambda$disablePostprocessFormattingInside$0(PostprocessReformattingAspect.java:113)
	at com.intellij.psi.impl.source.PostprocessReformattingAspect.disablePostprocessFormattingInside(PostprocessReformattingAspect.java:121)
	at com.intellij.psi.impl.source.PostprocessReformattingAspect.disablePostprocessFormattingInside(PostprocessReformattingAspect.java:112)
	at com.intellij.codeInsight.editorActions.EnterHandler.executeWriteAction(EnterHandler.java:78)
	at com.intellij.openapi.editor.actionSystem.EditorWriteActionHandler$1.run(EditorWriteActionHandler.java:57)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:905)
	at com.intellij.openapi.editor.actionSystem.EditorWriteActionHandler.doExecute(EditorWriteActionHandler.java:70)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.execute(EditorActionHandler.java:222)
	at com.intellij.codeInsight.template.impl.editorActions.EnterHandler.executeWriteAction(EnterHandler.java:49)
	at com.intellij.openapi.editor.actionSystem.EditorWriteActionHandler$1.run(EditorWriteActionHandler.java:57)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:905)
	at com.intellij.openapi.editor.actionSystem.EditorWriteActionHandler.doExecute(EditorWriteActionHandler.java:70)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler$3$1.perform(EditorActionHandler.java:206)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.doIfEnabled(EditorActionHandler.java:109)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.access$000(EditorActionHandler.java:38)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler$3.perform(EditorActionHandler.java:203)
	at com.intellij.openapi.editor.impl.CaretModelImpl.lambda$runForEachCaret$2(CaretModelImpl.java:348)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:437)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:340)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:331)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.execute(EditorActionHandler.java:200)
	at com.intellij.openapi.editor.actionSystem.EditorAction.lambda$actionPerformed$0(EditorAction.java:100)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:129)
	at com.intellij.openapi.editor.actionSystem.EditorAction.actionPerformed(EditorAction.java:109)
	at com.intellij.openapi.editor.actionSystem.EditorAction.actionPerformed(EditorAction.java:88)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:197)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:213)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:587)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$processAction$2(IdeKeyEventDispatcher.java:634)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:199)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:633)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:477)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:211)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:677)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:385)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

This code changes seems to reproduce bug. Initial code:
```
fn is_working_dir(dir: fs::ReadDir) -> bool {
    dir.any(|el| el == "./data")
}
```

I change code to this:
```
fn is_working_dir(dir: fs::ReadDir) -> bool {
    dir.any(|el| match el {(cursor__here)== "./data")
}
```

Then I hit "Enter" and error happens. Plugin version is `0.0.1.843`.

Great plugin guys! Keep up the good work!

--- a/src/main/kotlin/org/rust/ide/formatter/impl/alignment.kt
+++ b/src/main/kotlin/org/rust/ide/formatter/impl/alignment.kt
@@ -21,15 +21,15 @@ fun RustFmtBlock.getAlignmentStrategy(): RustAlignmentStrategy = when (node.elem
                 //          ))
                 // because due to applied alignment, closing paren has to be
                 // at least in the same column as the opening one.
+                var result = true
                 if (parent != null) {
                     val lBrace = parent.firstChildNode
                     val rBrace = parent.lastChildNode
-                    assert(lBrace.isBlockDelim(parent))
-                    assert(rBrace.isBlockDelim(parent))
-                    child.treeNonWSPrev() != lBrace || child.treeNonWSNext() != rBrace
-                } else {
-                    true
+                    if (lBrace.isBlockDelim(parent) && rBrace.isBlockDelim(parent)) {
+                        result = child.treeNonWSPrev() != lBrace || child.treeNonWSNext() != rBrace
+                    }
                 }
+                result
             }
             .alignUnlessBlockDelim()
             .alignIf(ctx.commonSettings.ALIGN_MULTILINE_PARAMETERS_IN_CALLS)

